app/models/group.rb

```diff
@@ -341,6 +348,20 @@ def remove_letter_logo=(deletion_param)
     end
   end
 
+  def name
```

Comment:
Hmm ist das eine gute Idee, die accessors für das `name` Attribut zu überschreiben?
Wegen dem "Principle of least surprise" fände ich es besser, stattdessen  `display_name` und `to_s` zu erweitern.

Reply:
ich glaube wenn wir sicherstellen wollen das das attribut auch den gleichen Wert via API zurück gibt, dann müssen wir das so machen, nicht?